numpy/core/_methods.py

```diff
@@ -22,17 +22,17 @@
 
 # avoid keyword arguments to speed up parsing, saves about 15%-20% for very
 # small reductions
-def _amax(a, axis=None, out=None, keepdims=False):
-    return umr_maximum(a, axis, None, out, keepdims)
+def _amax(a, axis=None, out=None, keepdims=False, initializer=None):
```

Comment:
Oh, the easy fix here is just to write `initializer=um.maximum.identity`, and avoid having `None` mean default

Reply:
Would that potentially break duck arrays? Passing `initializer` when it's not supposed to be passed?

Reply:
It might already be handling this down below.

```
if (initializer != Py_None && PyArray_ISOBJECT(arr) && PyArray_SIZE(arr) != 0) {
        Py_DECREF(initializer);
        initializer = Py_None;
        Py_INCREF(initializer);
    }
```

Reply:
It would affect `__array_ufunc__`, I suppose.

I think there might be a doc section on that that will need updating too.

Reply:
Oh, wait, it wouldn't. I retract my objection, those are handled in `fromnumeric.py`